examples/wip/viewport-animation-3d-heatmap/app.js

```diff
@@ -73,7 +73,7 @@ class Root extends Component {
       <AnimationMapController
         {...viewport}
         onViewportChange={this._onViewportChange.bind(this)}
-        animateViewport={true}>
+        viewportAnimationDuration={2000}>
```

Comment:
That is a long prop name. It will of course be discussed in API audit, but I suspect `duration` or `animationDuration` would work well?